ui/src/components/Logs/index.tsx

```diff
@@ -0,0 +1,204 @@
+import {
+  CaretSortIcon,
+  ClockIcon,
+  CrossCircledIcon,
+  ExclamationTriangleIcon,
+  InfoCircledIcon,
+  UpdateIcon,
+} from "@radix-ui/react-icons";
+import {
+  ColumnDef,
+  ColumnFiltersState,
+  SortingState,
+  VisibilityState,
+  flexRender,
+  getCoreRowModel,
+  getFilteredRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import { useMemo, useState } from "react";
+
+import {
+  DropdownMenu,
+  DropdownMenuCheckboxItem,
+  DropdownMenuContent,
+  DropdownMenuTrigger,
+  Button,
+  Input,
+} from "@flow/components";
+import { useT } from "@flow/lib/i18n";
+
+import { Table, TableBody, TableCell, TableRow } from "../Table";
+
+interface LogProps<TData, TValue> {
+  columns: ColumnDef<TData, TValue>[];
+  data: TData[];
+  selectColumns?: boolean;
+  showFiltering?: boolean;
+}
+
+type STATUS = "WARNING" | "INFO" | "ERROR";
```

Comment:
If you can create @flow/types/logs.ts and add this kind of type there 🙏🏻

Reply:
It's not final yet, will change most likely so I would like to keep it here for now. WIll depend on the API implementation actually

Reply:
Many types are not completely final yet, but I'm trying to keep the bigger types in the @flow/types dir. 
Something like logs, especially since the types will probably be needed in more than one place (debug log and run log) it feels better to be here.